Harvest.Db/dbo/Tables/Transfers.sql

```diff
@@ -1,21 +1,21 @@
-﻿CREATE TABLE [dbo].[Transfers] (
-    [Id]            INT             IDENTITY (1, 1) NOT NULL,
-    [Amount]        DECIMAL (18, 2) NOT NULL,
-    [Description]   NVARCHAR (40)   NULL,
-    [FromAccountId] INT             NOT NULL,
-    [ToAccountId]   INT             NOT NULL,
+CREATE TABLE [dbo].[Transfers] (
+    [Id]        INT             IDENTITY (1, 1) NOT NULL,
+    [Total]     DECIMAL (18, 2) NOT NULL,

```

Comment:
What's the reason for renaming the column `Amount` to `Total`?

Reply:
@sprucely It can roll several expense records into this total grouped by account.